src/test/java/org/swisspush/redisques/RedisQuesTest.java

```diff
@@ -349,6 +354,7 @@ public void getQueuesCountFilteredInvalidPattern(TestContext context) {
                 context.assertEquals(OK, message.result().body().getString(STATUS));
                 asyncEnqueue.countDown();
             });
+            Thread.sleep(10);
```

Comment:
Why is this necessary? The test will wait until the countDown has gone from 50 to 0